### DISCUSSION TITLE
Feature request: add support for managing recording devices as well

### DISCUSSION THREAD

**Discussion by cfischer69** [File-New-Project/EarTrumpet#651] [Ideas]
The playback device functionality is great.  I'd love to have the same capabilities for recording devices as well.  I have several and often need to switch between them and/or blend them.

**Comment by mortenlerudjordet** [comment]
This would be super nice

**Comment by karisinkko** [comment]
Yes, this would be logical.

**Comment by NathanRTracy** [comment]
I would love to have this feature, as well. That would make this be a complete audio interface manager.

**Comment by berghtho** [comment]
Would to be able to replace Audio Switcher with this.

**Comment by ICDeadPpl** [comment]
This would be great!

**Comment by narspt** [comment]
+1

**Comment by vertigo220** [comment]
Appears to be duplicate of #99
